models/aged_debtor.json

```diff
@@ -1,7 +1,8 @@
 {
     "fact_table": "aged_debtor_facts",
     "label": "Aged Debtor Analysis",
-    "description": "This is based on the AD and ADA return forms.",
+    "description": "Analysis of debtors by age of debt.",
```

Comment:
Worth adding "This does not list individual creditors, but rather the amount of debt in each age group for each line item category"?